cmd/lncli/cmd_payments.go

```diff
@@ -1123,16 +1174,22 @@ func queryRoutes(ctx *cli.Context) error {
 		}
 	}
 
+	blindedRoutes, err := parseBlindedPaymentParameters(ctx)
+	if err != nil {
+		return err
+	}
+
 	req := &lnrpc.QueryRoutesRequest{
-		PubKey:            dest,
-		Amt:               amt,
-		FeeLimit:          feeLimit,
-		FinalCltvDelta:    int32(ctx.Int("final_cltv_delta")),
-		UseMissionControl: ctx.Bool("use_mc"),
-		CltvLimit:         uint32(ctx.Uint64(cltvLimitFlag.Name)),
-		OutgoingChanId:    ctx.Uint64("outgoing_chan_id"),
-		TimePref:          ctx.Float64(timePrefFlag.Name),
-		IgnoredPairs:      ignoredPairs,
+		PubKey:              dest,
+		Amt:                 amt,
+		FeeLimit:            feeLimit,
```

Comment:
should we check this is not set for blinded routes?

Reply:
This logic should apply as before for blinded paths?

Reply:
exactly this feelimit still needs to be in place for blinded routes as well.